src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1877,7 +1880,7 @@ To create/register a plugin, you have to :
 	#define USES_P119	// BME680
 	#define USES_P120	// Thermocouple
 	#define USES_P121	// Candle
-//	   #define USES_P122	// NeoPixel       (MERGED?)
+  #define USES_P122	// SHT2x
```

Comment:
This is in the `PLUGIN_SET_EXPERIMENTAL` set, but that's no longer in use (though it used to be used for PluginPlayground plugins), so better not to be changed here, as these plugin IDs are referring to playground-IDs